packages/buckets/src/api/index.ts

```diff
@@ -55,51 +55,84 @@ export interface PushPathResult {
   root: string
 }
 
-export class BucketsGrpcClient {
-  public serviceHost: string
-  public rpcOptions: grpc.RpcOptions
-  /**
-   * Creates a new gRPC client instance for accessing the Textile Buckets API.
-   * @param context The context to use for interacting with the APIs. Can be modified later.
-   */
-  constructor(public context: ContextInterface = new Context(), debug = false) {
-    this.serviceHost = context.host
-    this.rpcOptions = {
-      transport: WebsocketTransport(),
-      debug,
-    }
+/**
+ * Response from bucket links query.
```

Comment:
I know is a pain to maintain these outside the gRPC types, but wow does this improve the DX considerably. Too bad there isn't a way to derive these automatically from the gRPC responses?